.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: java
+
+jdk:
+- oraclejdk7
+
+script:
+- mvn test -D-org.login.url="https://na14.salesforce.com" -D-org.username=$USERNAME -D-org.password=$PASSWORD -D-org.client.id=$CLIENT_ID -D-org.client.secret=$CLIENT_SECRET -D-org.wide.code.coverage.threshold=40 -D-team.code.coverage.threshold=40 -D-regex.for.selecting.source.classes.for.code.coverage.computation=HelloWorld -D-regex.for.selecting.test.classes.to.execute=HelloWord* -D-manifest.files.with.test.class.names.to.execute=ManifestFile_For_Unit_Test_Classes.txt -D-manifest.files.with.source.class.names.for.code.coverage.computation=ManifestFile.txt -D-max.test.execution.time.threshold=10
+
+env:
+  global:
+  - secure: ELavjRwFdNq/u+r/3q8CSI8YwpR3qAh2utkiUE43HjxIcxFhb16Y8yyZtqbdrHNrXHuXtTg6NPFo751I/08cfwlHaNLlhCiUc1mIOZGir0Y3qyJb2Idyu2SWDFU6Qok4qLuZLzCuizg9YOT2D0f7C3GcmYiSk2STg0LoZXt+4rxTHMX+qaQnzdN58o6dfstkMWFF402bh49QJmv6z0kfR6Jyxq40H2mEHrCVvA4UWE4kHbEk2GULui9mg4SKi9yx5yBrRhuAJcJsZq8aIwuApRgLuZdh5iBWUy1M9TLaFL7izC/mV+7j40k4fcOxaWjsP6D+2OnqMmi2p1v8M1l4cT6YuJGRx9MVDtNvC8jkQObWJF2+OuaFFtwZOIUlzscMNCh0PwrlsQgZQgzssYE8yOkiuRKNQq9+3W42+ZH+7iGzvu1O23JcSjhff7FKIWkmDlZhR55o0vtQS2M4uBVRWpgurMBPHcTgi72pzxEkKUNA1Bu0zAQSWMSwd7/3Z04fyCWbanJFm+gaPvPqBZlGtnwW1SyOBrgknvDDvlcgqAoKRC8M4VeYLxYlbsHIrLVPInWZ6KOv+6TPISoqOM/3AQPxUNcAuUTZnfRKNlKCB5kQxu3g4tcwARc47ZNhUSVX+XBtFVdDqlx2PrbDMQIWAMb1fAS8010jOcL2z3uE3j0=
```

Comment:
The recommended way by travis to encrypt secrets is by using 'secure' global env variables as given [here](https://docs.travis-ci.com/user/encryption-keys/). So followed the same method.